heron/tools/cli/src/python/args.py

```diff
@@ -151,6 +154,32 @@ def add_extra_launch_classpath(parser):
   '''
   parser.add_argument(
       '--extra-launch-classpath',
-      metavar='(a string; additional JVM class path for launching topology)',
-      default="")
+      metavar='CLASS_PATH',
+      default="",
+      help='Additional JVM class path for launching topology')
+  return parser
+
+def add_dry_run(parser):
```

Comment:
How is the case handled where someone tries to dry-run a command that doesn't support it? How does the user know which command dry run is valid for? For example:
```
heron deactivate --dry-run
```